src/storage/mvcc/txn.rs

```diff
@@ -270,7 +270,7 @@ impl MvccTxn {
         Ok(())
     }
 
-    pub fn gc(&mut self, key: &Key, safe_point: u64) -> Result<()> {
+    pub fn gc(&mut self, key: &Key, safe_point: u64) -> Result<(usize, usize)> {
```

Comment:
The return type will be fairly self-documenting if this switches from a tuple to a struct `GcInfo { versions: usize, delete_versions: usize }`